docs/backend-requests/resources/session-tokens.mdx

```diff
@@ -18,13 +18,13 @@ Below are the default session claims that Clerk generates for you:
 * `nbf`: not before - the time before which the token is considered invalid, as a Unix timestamp. Determined using the **Allowed Clock Skew** JWT template setting in the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=jwt-templates).
 * `sid`: session ID - the ID of the current session (e.g. `sess_123`).
 * `sub`: subject - the ID of the current user of the session (e.g. `user_123`).
+* `act`: actor - will only be included if the user is impersonating another user. See [user impersonation](/docs/custom-flows/user-impersonation#jwt-claims) for more information.
```

Comment:
```suggestion
* `act`: actor - will only be included if the user is impersonating another user. See [user impersonation](/docs/custom-flows/user-impersonation) for more information. 
```
We should remove this anchor, unless it's intentional 👍. When I clicked this link, I expected to be at the top of the page